Scripts/script-QRadarGetOffenseCorrelations.yml

```diff
@@ -0,0 +1,91 @@
+commonfields:
+  id: QRadarGetOffenseCorrelations
+  version: -1
+name: QRadarGetOffenseCorrelations
+script: |-
+  QUERY = "SELECT *,\"CRE Name\",\"CRE Description\",CATEGORYNAME(highlevelcategory) FROM events WHERE \"CRE NAME\" <> NULL AND INOFFENSE({0}) START '{1}'"
+
+  d_args = demisto.args()
+
+  offense_id = demisto.get(d_args, 'offenseID')
+  start_time = demisto.get(d_args, 'startTime')
+
+  d_args["query_expression"] = QUERY.format(offense_id,start_time)
+
+  resp = demisto.executeCommand('QRadarFullSearch',d_args)
+  if isError(resp[0]):
+      demisto.results(resp)
+  else:
+      data = demisto.get(resp[0],'Contents.events')
+
+      if not data:
+          resp[0]['HumanReadable'] = "No Correlations were found for offense id {0}".format(offense_id)
+      else:
+          data = data if isinstance(data,list) else [data]
+
+          context = {}
+          context['Correlation'] = []
+
+          for corr in data:
+              correlation = {}
+              correlation["SourceIP"] = demisto.get(corr,"sourceip")
+              correlation["CREDescription"] = demisto.get(corr,"CRE Description")
+              correlation["CREName"] = demisto.get(corr,"CRE Name")
+              correlation["QID"] = demisto.get(corr,"qid")
+              correlation["DestinationIP"] = demisto.get(corr,"destinationip")
+              correlation["Category"] = demisto.get(corr,"categoryname_highlevelcategory")
+              correlation["Username"] = demisto.get(corr,"username")
+              correlation["StartTime"] = demisto.get(corr,"starttime")
+
+              context['Correlation'].append(correlation)
+
+          resp[0]['EntryContext'] = context
+
+  demisto.results(resp)
+
+type: python
+tags:
+- QRadar
+comment: Return the QRadar offense correlations if exist in logs.
+system: true
+args:
+- name: offenseID
+  required: true
+  default: true
+  description: The offense ID to query, can be found in label under ${incident.labels.id}
```

Comment:
use default value
And see rest of places you suggested to look at a specific location